chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -309,6 +312,10 @@ def cinder_volume
           @node[:block_device][@device][:vendor] == "cinder" && @node[:block_device][@device][:model] =~ /^volume-/
         end
 
+        def mtu
+          @node[:mtu] || 1500
```

Comment:
Is the `||` necessary? In `initialize` we test for a non-existing value already.